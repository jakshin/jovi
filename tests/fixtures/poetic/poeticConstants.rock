Something is null
Say Something

My intentions are true
Say my intentions

The number was 1
Say the number

Things were "good enough, for now"
Say Things
