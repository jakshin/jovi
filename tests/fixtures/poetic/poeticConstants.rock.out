null
true
1
good enough, for now
